docs/api/role-management/put.asciidoc

```diff
@@ -54,22 +84,60 @@ PUT /api/security/role/my_kibana_role
       "query" : "{\"match\": {\"title\": \"foo\"}}"
     } ]
   },
-  "kibana": {
-    "global": ["all"]
-  }
+  "kibana": [
+    {
+      "base": [
+        "read"
+      ],
+      "feature": {
+        "advancedSettings": ["all"]
+      },
+      "spaces": [
+        "*"
+      ]
+    }
+  ]
 }
 --------------------------------------------------
 // KIBANA
 
-==== Response
-
-A successful call returns a response code of `204` and no response body.
+===== Example 2
+Granting "dashboard only" access to only the Marketing space.
 
+[source,js]
+--------------------------------------------------
+PUT /api/security/role/my_kibana_role
+{
+  "metadata" : {
+    "version" : 1
+  },
+  "elasticsearch": {
```

Comment:
The "elasticsearch" section here might detract from the primary purpose of illustrating that you don't have to assign the `base` privilege.

Perhaps a separate example showing that you can grant Kibana and Elasticsearch privileges at the same time would be clearer?